fatips/102.md

```diff
@@ -14,93 +14,66 @@ tokens.
 
-#### Signed Data
+Entries in the index chain represent individual FAT tokens being indexed. Each entry is a simple pointer containing the necessary data to find the token's chain, where user facing data such as Name, Symbol & etc. can be found.
 
-`data` specified as per [FATIP-101](101) is derived as follows:
+From [FATIP-101](101.md), the `data` being signed shall be the Chain ID of the token chain concatenated with the index entry content. The `issuer` signing the content shall be the hex encoded Identity Root Chain ID specified in the in the entry JSON, and the `IDKey` used shall be the IDKey of `issuer` at the time of entry to Factom.
 
-```
-idNonce = sha256(tokenid + salt)
-```
 
-
-### Index Entry Content Example
+### Token Index Entry Content Example
 
 ```json
  {
-  "tokenID": "mytoken",
-  "salt": "874220a808090fb736f345dd5d67ac26eab94c9c9f51b708b05cdc4d42f65aae",
-  "idNonce": "cc4fee39dcdcfc1999ab07689230321acdd83fcd0ace521107041ef354b9cfb5",
-  "idSignature": "33467893a440561d96ae27798dc8be291e1ce264d3c6f36f33a0d983e745f1d87db61c77946fe57db3e185f548d51da85106dfec592383a556091dd45f384b0c"
+  "id": "mytoken",
+  "issuer":"888888dd9e60c1f0216f753caf5c9b5be4c9ca69db27a6c33d30dce3fe5ee709",
+  "salt": "874220a808090fb736f345dd5d67ac26eab94c9c9f51b708b05cdc4d42f65aae"
 }
 ```
 
 
-### Index Entry Fields Summary & Validation
-
-| Field       | Type   | Description                           | Validation                                                  | Required |
-| ----------- | ------ | ------------------------------------- | ----------------------------------------------------------- | -------- |
-| tokenID     | string | The ID of the FAT token being indexed | FAT Token Exists                                            | Y        |
-| salt        | string | Random String                         |                                                             | N        |
-| idNonce     | string | Nonce for issuing identity to sign    |                                                             | Y        |
-| idSignature | string | ed25519 signature of `idNonce`        | ed25519 signature validation of sha256d(`tokenID` + `salt`) | Y        |
-
-
-## Validation
+### Token Index Entry Fields Summary & Validation
 
-Index entries must follow a simple set of validation rules in addition to those
-the above table:
+| Field  | Type   | Description                           | Validation                                       | Required |
+| ------ | ------ | ------------------------------------- | ------------------------------------------------ | -------- |
+| id     | string | The ID of the FAT token being indexed | FAT Token Exists                                 | Y        |
+| issuer | string | The root Chain ID of the issuer       | Chain must exist, must be valid Factom identity. | Y        |
+| salt   | string | Random string to salt the content     |                                                  | N        |
```

Comment:
In which case would a salt ever be needed?

Reply:
Put it on the copy pasta? :spaghetti: _heh_
Jokes aside I think we're safe from replay here, `salt` can be omitted.

Reply:
Resolved in commit [142f059](https://github.com/Factom-Asset-Tokens/FAT/pull/23/commits/142f059e100ed6a6e33674e6d91a108a0a45a9ea)